<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DropPox</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="agregarArchivoButton">
      <property name="text">
       <string>Agregar Archivo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="agregarCarpetaButton">
      <property name="text">
       <string>Agregar Carpeta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="descargarButton">
      <property name="text">
       <string>Descargar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actualizarButton">
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="volverButton">
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="archivoslistWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Carpetas observadas</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
